os/debian/Dockerfile.debian

```diff
@@ -127,9 +137,27 @@ RUN curl -1sLf 'https://dl.cloudsmith.io/public/cloudposse/packages/cfg/setup/ba
 RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
 RUN curl -1sLf 'https://packages.cloud.google.com/apt/doc/apt-key.gpg' | APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=true apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
 
-RUN apt-get update && apt-get install -y \
-    $(grep -h -v '^#' /etc/apt/packages.txt /etc/apt/packages-debian.txt | sed -E 's/@(cloudposse|community|testing)//g' ) && \
-    mkdir -p /etc/bash_completion.d/ /etc/profile.d/ /conf && \
+# If runnig Dokcer build on Macos m1:
+# Since the packages are not available for arm64 let's build them using sources from Cloudposse's `package` repository
+RUN if [ "$ARCH" = "arm64" ] ; then \
+        # Install commonly known packages from public Debian repo
+        apt-get update && apt-get install -y $(grep -h -v '^#' /etc/apt/packages.txt /etc/apt/packages-debian.txt | sed -E '/@(cloudposse|testing)/d' | sed -E 's/@community//g') && \
+        git clone https://github.com/cloudposse/packages.git && \
+        cd packages && \
+        # Packages with `@cloudposse` or `@testing` annotation have to be built from source
+        make -C install $(grep -h -v '^#' /etc/apt/packages.txt /etc/apt/packages-debian.txt | sed -E '/_amd64/d' | sed -nE 's/@(cloudposse|testing)//pg') && \
+        # # Packages ending with _amd64 have no arm64 equivalent
+        make -C install $(grep -h -v '^#' /etc/apt/packages.txt /etc/apt/packages-debian.txt | sed -nE 's/[@]?[a-zA-Z0-9]*_amd64//pg') ARCH=amd64 && \
+        # The `-slim` version of gomplate is not available for ARM 64 so we have to alter the DOWNLOAD_URL as the one in
+        # `packages` repo is not compatible.
+        make -C install gomplate DOWNLOAD_URL='$(PACKAGE_REPO_URL)/releases/download/v$(PACKAGE_VERSION)/$(PACKAGE_NAME)_$(OS)-$(ARCH)' ; \
+    # On any other architecture proceed as usual. Packages will come from `https://dl.cloudsmith.io/public/cloudposse/packages/deb/debian bullseye main` repo.
+    else \
+        apt-get update && apt-get install -y \
+            $(grep -h -v '^#' /etc/apt/packages.txt /etc/apt/packages-debian.txt | sed -E 's/@(cloudposse|community|testing).*//g' ) ; \
+    fi
+
+RUN mkdir -p /etc/bash_completion.d/ /etc/profile.d/ /conf && \
```

Comment:
```suggestion
# Since the packages are not available for arm64 let's build them using sources from Cloudposse's `package` repository
RUN if [ "${TARGETARCH}" = "arm64" ] ; then \
        # Install commonly known packages from public Debian repo
        apt-get update && apt-get install -y $(grep -h -v '^#' /etc/apt/packages.txt /etc/apt/packages-debian.txt | sed -E '/@(cloudposse|testing)/d' | sed -E 's/@community//g') && \
        git clone https://github.com/cloudposse/packages.git && \
        cd packages && \
        # Packages with `@cloudposse` or `@testing` annotation have to be built from source
        make -C install $(grep -h -v '^#' /etc/apt/packages.txt /etc/apt/packages-debian.txt | sed -E '/_amd64/d' | sed -nE 's/@(cloudposse|testing)//pg') && \
        # # Packages ending with _amd64 have no arm64 equivalent
        make -C install $(grep -h -v '^#' /etc/apt/packages.txt /etc/apt/packages-debian.txt | sed -nE 's/[@]?[a-zA-Z0-9]*_amd64//pg') ARCH=amd64 && \
        # The `-slim` version of gomplate is not available for arm64 so we have to alter the DOWNLOAD_URL as the one in
        # `packages` repo is not compatible.
        make -C install gomplate DOWNLOAD_URL='$(PACKAGE_REPO_URL)/releases/download/v$(PACKAGE_VERSION)/$(PACKAGE_NAME)_$(OS)-$(TARGETARCH)' ; \
    # On any other architecture proceed as usual. Packages will come from `https://dl.cloudsmith.io/public/cloudposse/packages/deb/debian bullseye main` repo.
    else \
        apt-get update && apt-get install -y \
            $(grep -h -v '^#' /etc/apt/packages.txt /etc/apt/packages-debian.txt | sed -E 's/@(cloudposse|community|testing).*//g' ) ; \
    fi

RUN mkdir -p /etc/bash_completion.d/ /etc/profile.d/ /conf && \
```